resources/istio/values.yaml

```diff
@@ -41,15 +41,19 @@ meshConfig:
   accessLogFile: /dev/stdout
   accessLogEncoding: JSON
   trustDomain: cluster.local
+  defaultProviders:
+    tracing: []
   defaultConfig:
     holdApplicationUntilProxyStarts: true
-    tracing:
-      sampling: 1
-      zipkin:
-        address: "zipkin.kyma-system:9411"
   enablePrometheusMerge: false
   enableTracing: "{{ .Values.global.tracing.enabled }}"
   extensionProviders:
+    - name: kyma-traces
+      opencensus:
+        service: "telemetry-trace-collector-internal.kyma-system.svc.cluster.local"
```

Comment:
should we pass this also from the values yaml ? I am afraid if we update the service name then we might forget to change it here

Reply:
and if it is in the values.yaml of the istio component you will not forget about it?

Reply:
This is values.yaml ;-)